diff --git a/bfd/coff-sh.c b/bfd/coff-sh.c
index 10d203f..494a5f2 100644
--- a/bfd/coff-sh.c
+++ b/bfd/coff-sh.c
@@ -1392,13 +1392,13 @@ sh_relax_delete_bytes (bfd *abfd,
 	{
 	  struct internal_syment sym;
 
-#ifdef COFF_WITH_PE
-	  if (irelscan->r_type != R_SH_IMM32
-	      && irelscan->r_type != R_SH_IMAGEBASE
-	      && irelscan->r_type != R_SH_IMM32CE)
-#else
+
+
+
+
+
 	  if (irelscan->r_type != R_SH_IMM32)
-#endif
+
 	    continue;
 
 	  bfd_coff_swap_sym_in (abfd,
diff --git a/zlib/adler32.c b/zlib/adler32.c
index b3d35e2..af0429d 100644
--- a/zlib/adler32.c
+++ b/zlib/adler32.c
@@ -1,3 +1,6 @@
+#ifdef FRCOV
+#include <stdio.h>
+#endif
 /* adler32.c -- compute the Adler-32 checksum of a data stream
  * Copyright (C) 1995-2011, 2016 Mark Adler
  * For conditions of distribution and use, see copyright notice in zlib.h
diff --git a/zlib/crc32.c b/zlib/crc32.c
index affcd46..72b2454 100644
--- a/zlib/crc32.c
+++ b/zlib/crc32.c
@@ -1,3 +1,6 @@
+#ifdef FRCOV
+#include <stdio.h>
+#endif
 /* crc32.c -- compute the CRC-32 of a data stream
  * Copyright (C) 1995-2006, 2010, 2011, 2012, 2016 Mark Adler
  * For conditions of distribution and use, see copyright notice in zlib.h
diff --git a/zlib/deflate.c b/zlib/deflate.c
index 586cd15..4dd325d 100644
--- a/zlib/deflate.c
+++ b/zlib/deflate.c
@@ -1,3 +1,6 @@
+#ifdef FRCOV
+#include <stdio.h>
+#endif
 /* deflate.c -- compress data using the deflation algorithm
  * Copyright (C) 1995-2017 Jean-loup Gailly and Mark Adler
  * For conditions of distribution and use, see copyright notice in zlib.h
diff --git a/zlib/inflate.c b/zlib/inflate.c
index ac333e8..9e4cfed 100644
--- a/zlib/inflate.c
+++ b/zlib/inflate.c
@@ -1,3 +1,6 @@
+#ifdef FRCOV
+#include <stdio.h>
+#endif
 /* inflate.c -- zlib decompression
  * Copyright (C) 1995-2016 Mark Adler
  * For conditions of distribution and use, see copyright notice in zlib.h
diff --git a/zlib/inftrees.c b/zlib/inftrees.c
index 2ea08fc..a3bf735 100644
--- a/zlib/inftrees.c
+++ b/zlib/inftrees.c
@@ -1,3 +1,6 @@
+#ifdef FRCOV
+#include <stdio.h>
+#endif
 /* inftrees.c -- generate Huffman trees for efficient decoding
  * Copyright (C) 1995-2017 Mark Adler
  * For conditions of distribution and use, see copyright notice in zlib.h
